docs/literate.jl

```diff
@@ -11,6 +11,10 @@ using Pkg: Pkg
 const EXAMPLEPATH = joinpath(@__DIR__, "..", "examples", EXAMPLE)
 Pkg.activate(EXAMPLEPATH)
 Pkg.instantiate()
+io = IOBuffer()
+Pkg.status(; io=io)
```

Comment:
As mentioned in the issue, it is not sufficient to just print the status of the direct dependencies. If one wants to rerun the example/tutorial with the same setup one has to know also the Julia version and the status of the indirect dependencies (it might still not possible due to differences of binary dependencies between different OSes and architectures but probably that's less important). The status of all packages one could obtain with
```suggestion
Pkg.status(; io=io, mode=Pkg.PKGMODE_MANIFEST)
```

You can also check https://github.com/SciML/SciMLTutorials.jl/blob/e600166d1f062411e8a4882faa318f436296042d/src/SciMLTutorials.jl#L83-L127 for more inspiration. In particular, it might be nice to print both the status of only the direct dependencies and of all dependencies, and it would be easier to use `sprint` instead of fiddling with `IOBuffer`.

Reply:
We are aware of this, but we rather have this then deal with Pkg version incompabilities every change of version in the main package.
I added some of your suggestions. The SciML tutorial is a really good inspiration

Reply:
Sure, I don't object to this change, just try to improve it. I'm annoyed by the updates as well :stuck_out_tongue: 

Maybe add something like https://github.com/SciML/SciMLTutorials.jl/blob/e600166d1f062411e8a4882faa318f436296042d/src/SciMLTutorials.jl#L100-L101 as well?